example.env

```diff
@@ -0,0 +1,15 @@
+DOCKER_GIT_CREDENTIALS=https://{user}:{pw}@github.com
```

Comment:
@mykelswitzer this is missing a lot of documentation and extra example variables that a lot of the example files have

Reply:
@amarine7882 just to note the progenitor tool doesnt generate any env files, example or otherwise. So I added this manually. But I can copy over one from another project that has all the comments/extra vars

Reply:
ahh. Contracts is a good example then
https://github.com/caring/contracts/blob/master/.example.env